apps/cfp/tasks.py

```diff
@@ -150,3 +150,31 @@ def create_tags(tags_to_create):
 
     db.session.commit()
     app.logger.info(f"Successfully created {tags_created} new tags.")
+
+
+@cfp.cli.command(
+    "delete_tags",
+    help=f"Delete tags from the Database. Any tagged proposals will have the tags removed.",
```

Comment:
I guess "Any tagged proposals will have the tags removed" was a first draft?

Reply:
no, the tags have got the correct cascade setting so deleting the tag removes it from the list (at least that's what local testing did)

Reply:
Ahh, I read a `return` that isn't there. Maybe it should say something like f"'{tag_name}' will be removed from the following proposals: {tagged_proposals}".

Reply:
Good point, :172 is now? `f"'{tag_name}' will be removed from the proposals with ids: {tagged_proposals}"`